src/pages/onboarding/Onboarding.vue.js

```diff
@@ -0,0 +1,97 @@
+import { swiper, swiperSlide } from 'vue-awesome-swiper'
+import router from '@/router'
+
+export default {
+  name: 'intro',
+  data () {
+    const vue = this
+    return {
+      colors: ['#C783FF', '#50E3C2', '#90DBFF', '#FF5537'],
+      swiperOption: {
+        notNextTick: true,
+        grabCursor: true,
+        setWrapperSize: true,
+        parallax: true,
+        pagination: '.swiper-pagination',
+        paginationClickable: true,
+        prevButton: '.swiper-button-prev',
+        nextButton: '.swiper-button-next',
+        mousewheelControl: true,
+        onTransitionStart(swiper) {
+          vue.changeThemeColor(swiper)
+        },
+        onTransitionEnd(swiper) {
+          vue.handleSlideChange(swiper)
+        }
+      }
+    }
+  },
+  computed: {
+    onboardingBgElem() {
+      return document.querySelector('.onboarding-background')
+    },
+    onboardingWorldElem() {
+      return document.querySelector('.onboarding-world')
+    },
+    onboardingPaginationElem() {
+      return document.querySelector('.swiper-pagination')
+    },
+    onboardingBulletElems() {
+      return document.querySelectorAll('.swiper-pagination-bullet')
+    },
+    onboardingFinishElem() {
+      return document.querySelector('.swiper-button-finish')
+    }
+  },
+  methods: {
+
+    handleSlideChange(swiper) {
+      switch(swiper.activeIndex) {
+        case 2:
+          this.onboardingWorldElem.classList.add('active')
```

Comment:
How does this work? Shouldn't it be `this.onboardingWorldElem().classList.add('active')`? Right now it's calling `classList.add('active')` on a function instead of an element.